pkg/util/i18n/i18n.go

```diff
@@ -53,10 +53,6 @@ func loadSystemLanguage() string {
 		return "default"
 	}
 	pieces := strings.Split(langStr, ".")
-	if len(pieces) == 0 {
```

Comment:
Okay, I think I know what is happening here. Instead of `len(pieces)` perhaps this was supposed to be `len(pieces[0])` to detect zero character locales once the encoding was stripped?

Perhaps @brendandburns who wrote this can shed some light